appdaemon/README.md

```diff
@@ -1,4 +1,28 @@
 This appdaemon app allows rgb lights to be synced with the thumbnail of a media player in Home Assistant.
+# Prerequsites
```

Comment:
Typo
```suggestion
# Prerequisites
```

Reply:
Also I would like this whole section to be under `# Configuration` with heading-2 (`## Prerequisites`).